examples/LiftSim_demo/wrapper.py

```diff
@@ -0,0 +1,84 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# wrapper part modified from
+# https://github.com/openai/gym/blob/master/gym/core.py
+
+from rlschool import LiftSim
+from wrapper_utils import obs_dim, act_dim, mansion_state_preprocessing
+from wrapper_utils import action_idx_to_action
+
```

Comment:
All of the wrapper should be a subclass of `gym.Wrapper`. Please refer to what we did [before](http://gitlab.baidu.com/nlp-ol/R2_NIPS2018_ProstheticsChallenge_DDPG/blob/master/env_wrapper.py).

Reply:
The link raises 404 error

Reply:
![image](https://user-images.githubusercontent.com/25046619/62443268-08f42300-b78d-11e9-9a39-9437e9b35f25.png)
Oh, I forgot that you don't have the permission to view this page.